src/build.py

```diff
@@ -68,7 +68,11 @@ def render(template, dest, **kwargs):
     template = environment.get_template(template)
     text = template.render(**kwargs)
 
-    f = file(dest, 'wb')
+    if sys.version_info[0] < 3:
+        f = file(dest, "wb")
+    else:
+        f = open(dest, 'wb')
+
```

Comment:
just wondering open works in 2 and 3 so why do we even need file ?
in fact we could probably do just these two lines ? but i am curious if there is a reason for file 

with open(dest, 'wb') as f:
    f.write(text.encode('utf-8'))